naughtystrings/naughtystrings.go

```diff
@@ -0,0 +1,37 @@
+// Package naughtystrings is a collection of strings that have a high probability of causing issues when used as user input.
+package naughtystrings
+
+import (
+	"encoding/json"
+	"os"
+	"path"
+	"runtime"
+)
+
+// Base64Encoded returns the strings encoded in base 64.
+func Base64Encoded() []string {
+	return strings("blns.base64.json")
+}
+
+// Unencoded returns the strings.
+func Unencoded() []string {
+	return strings("blns.json")
+}
+
+func strings(file string) []string {
+	var _, here, _, _ = runtime.Caller(0)
+
+	there, err := os.Open(path.Join(path.Dir(path.Dir(here)), file))
+
+	if err != nil {
+		panic(err)
```

Comment:
I think this excerpt from [Effective Go](https://golang.org/doc/effective_go.html#errors) applies here:

>This is only an example but real library functions should avoid panic. If the problem can be masked or worked around, it's always better to let things continue to run rather than taking down the whole program. One possible counterexample is during initialization: if the library truly cannot set itself up, it might be reasonable to panic, so to speak.